adopters/index.html

```diff
@@ -203,7 +203,7 @@
 	</section>
 
 	<footer id="footer">
-		The Contributor Covenant was created by <a href="http://where.coraline.codes/" title="Coraline Ada Ehmke">Coraline Ada Ehmke</a> in 2014 and is released under an <a href="https://github.com/ContributorCovenant/contributor_covenant/blob/master/LICENSE">MIT license</a>.<br>
+		The Contributor Covenant was created by <a href="http://where.coraline.codes/" title="Coraline Ada Ehmke">Coraline Ada Ehmke</a> in 2014 and is released under an <a href="https://github.com/ContributorCovenant/contributor_covenant/blob/master/LICENSE.md">MIT license</a>.<br>
```

Comment:
Ok. I will change `MIT license` to `Creative Commons Attribution 4.0 International Public License.`.